lib/icingadb/icingadb-itl.conf

```diff
@@ -0,0 +1,35 @@
+/* Icinga 2 | (c) 2022 Icinga GmbH | GPLv2+ */
+
+System.assert(Internal.run_with_activation_context(function() {
+	template CheckCommand "icingadb-check-command" use (checkFunc = Internal.IcingadbCheck) {
+		execute = checkFunc
+	}
+
+	object CheckCommand "icingadb" {
+		import "icingadb-check-command"
+
+		var criticalPendingSize = 1000000000
+		var objectSize = 1000
+		var maxObjectsPerQuery = 100
+		var additionalSmallerQueriesPerObject = 1
+		var criticalPendingQueries = (criticalPendingSize / (maxObjectsPerQuery * objectSize)) * (1 + additionalSmallerQueriesPerObject)
+
+		vars.icingadb_name = "icingadb"
+		vars.icingadb_downfor_warning = 10s
+		vars.icingadb_downfor_critical = 60s
+		vars.icingadb_heartbeat_warning = 10s
+		vars.icingadb_heartbeat_critical = 60s
+		vars.icingadb_idlefor_warning = 1.5m
+		vars.icingadb_idlefor_critical = 3m
+		vars.icingadb_history_backlog_warning = 1m
+		vars.icingadb_history_backlog_critical = 15m
+		vars.icingadb_pending_queries_warning = criticalPendingQueries / 2
+		vars.icingadb_pending_queries_critical = criticalPendingQueries
+		vars.icingadb_syncago_warning = 5 * 366d
+		vars.icingadb_syncago_critical = 10 * 366d
+		vars.icingadb_dumpago_warning = 5 * 366d
+		vars.icingadb_dumpago_critical = 10 * 366d
```

Comment:
I'd rather have no threshold set rather than some artificial high value that's probably never reached realistically. This way it just shows pointless thresholds in the web interface.